src/Symfony/Component/Secret/Command/SecretsDecryptToLocalCommand.php

```diff
@@ -9,21 +9,21 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Bundle\FrameworkBundle\Command;
+declare(strict_types=1);
+
+namespace Symfony\Component\Secret\Command;
 
-use Symfony\Bundle\FrameworkBundle\Secrets\AbstractVault;
 use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\ConsoleOutputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Secret\AbstractVault;
 
 /**
  * @author Nicolas Grekas <p@tchwork.com>
- *
- * @internal
```

Comment:
Most likely should be still there.

Reply:
Even if used in the FramworkBundle ?

Reply:
I also interpret it that no external project may rely on the interfaces of the class. In that case yes, usage in other libraries of the symfony package should work together with the internal tag.